app/assets/javascripts/markdown_editor.js

```diff
@@ -42,6 +63,26 @@ const initializeMarkdownEditors = () => {
           const content = toastEditor.getMarkdown();
           formInput.value = content;
         },
+        // 
+        keyup: (_, event) => {
+          if (event.keyCode === BACKTICK) {
```

Comment:
The [`keyCode` seems to be deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), and it is now suggested to use [`key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) or [`code`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code).

I was checking for suitable alternatives:
- The first one, `key` will just return `Dead`. Unfortunately, while that's correct, it is also returned for other dead keys on my keyboard (such as `^`). 
- The second one, `code` is better. It is returned as `Backquote`. However, unfortunately, that might depend on the keyboard layout (and partially the browser) as my own experiments and other sources suggested. (For my German keyboard layout, I always get a value of `Equal`.)

Some websites indicate that one should rather use `keypress`, but that is not exposed by ToastUi. That having said, none of the alternatives is satisfying in my alternative. Given the fact that even the deprecated `keyCode`s behave differently in browsers but still work magically, I would leave this code as is. 

If you want to test the behaviour yourself, use [this website](https://www.toptal.com/developers/keycode) and press keys. For me, hitting the backtick with aforementioned website gives:
- `187` in Chrome
- `192` in Firefox

Still, our fix is working in both browsers for me flawlessly, even when switching keyboard layouts. 🤷 

<hr>

Maybe we should add this as a comment, too?